modules/js/generator/embindgen.py

```diff
@@ -953,8 +955,11 @@ def gen(self, dst_file, src_files, core_bindings):
     headers = open(sys.argv[3], 'r').read().split(';')
     coreBindings = sys.argv[4]
     whiteListFile = sys.argv[5]
-    exec(open(whiteListFile).read())
```

Comment:
Old behavior seems broken.

Besides of json there are namespaces configuration were possible (see `namespace_prefix_override`), so python configuration code is preferable and provides full customization without unnecessary limitation of json.